configuration/mfa/mfa.md

```diff
@@ -0,0 +1,97 @@
+# Multi-Factor Authentication
+
+Gravwell supports Multi-Factor Authentication, also known as Two-Factor Authentication, for added security on local (non-SSO) accounts. When enabled, users must provide their password *and* a valid additional factor (either a [TOTP](https://en.wikipedia.org/wiki/Time-based_one-time_password) "authenticator app" code or a recovery code) to login.
+
+As of Gravwell 5.5.0, any user may choose to enable MFA for their account, or administrators may require it explicitly.
+
+## Requiring MFA
+
+To enable MFA, add the following line to the `[Global]` block of your webserver's `gravwell.conf`:
```

Comment:
Perhaps, "to require MFA"  might work better here? I think "enabling" sounds a bit misleading in this context.